### ISSUE TITLE
The image quality generated using the official website's pkl file is poor

### ISSUE THREAD

**Issue by 292533846** [NVlabs/stylegan2-ada-pytorch#287]
**Describe the bug**
After configuring the environment, using the test instructions in generate.py can compile successfully, but the effect of generating images is poor.

**To Reproduce**
1. in pycharm terminal  test  python generate.py --outdir=out --trunc=1 --seeds=85,265,297,849 --network=https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/metfaces.pkl
2. See error as the following img
![image](https://github.com/NVlabs/stylegan2-ada-pytorch/assets/132440197/8704bc97-1b20-487e-8777-09d739f3320e)

**Desktop (please complete the following information):**
 - OS: [ Windows 10]
 - PyTorch version (e.g., pytorch 1.8.0)
 - CUDA toolkit version (e.g., CUDA 11.1)
 - NVIDIA driver version
 - GPU [e.g., Titan V, GTX1650]
 - Docker: did you use Docker? No Docker

**Comment by 292533846** [comment]
@nurpax May I ask if you know what caused this? I have been searching in the community for a long time and there have been no similar problems. Thank you very much

**Comment by Great-Bucket** [comment]
Hi, one small thing I noticed from your post is that your command is generating --seeds=85,265,297,849.  But the images that you are showing in your 2. are numbered sequentially from seed 29 to 50.  Given that the image seeds and count are not consistent with the '--seeds=85,265,297,849' parameter - perhaps you are looking in the wrong folder for the results.  That doesn't explain why the images you showed look like noise, but I thought I'd point it out since sometimes the obvious issue is the one I overlook.  Best of luck!
